File: beakjoon_1110.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int num = n;
        int cnt = 0;

        while (true) { 
            int a = num / 10;
            int b = num % 10;
            int c = (a + b) % 10;
            num = (b * 10) + c;
            cnt += 1;
            
            if (n == num) {
                break;
            }
        }
        System.out.println(cnt);
    }
}

File: python/beakjoon_1110.py
import sys
input = sys.stdin.readline

n = int(input()) #26
num = n
cnt = 0

while True:
    a = num // 10 #2
    b = num % 10  #6
    c = (a + b) % 10   #8
    num = (b*10) + c
    cnt += 1
    
    if n == num:
        break

print(cnt)
